testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -33,6 +33,23 @@
 from MDAnalysisTests.util import no_deprecated_call
 
 
+class Test_Results:
```

Comment:
If you filter forbidden keys then you'll also need a test for it.

Reply:
Currrently, I don't know any that we forbid.